includes/class-amp-story-post-type.php

```diff
@@ -174,6 +174,8 @@ public static function register() {
 
 		add_action( 'enqueue_block_editor_assets', array( __CLASS__, 'enqueue_block_editor_scripts' ) );
 
+		add_action( 'enqueue_block_editor_assets', array( __CLASS__, 'export_latest_stories_block_editor_data' ), 100 );
```

Comment:
This inline script is not needed in the stories editor.

Reply:
Fixed in cc06129